### ISSUE TITLE
[bug] Multiple login methods impossible

### ISSUE THREAD

**Issue by Firesphere** [silverstripe/silverstripe-framework#6788]
In the template `Security_MultiAuthenticatorLogin`, the method h`$AuthenticatorName` is called, which returns null.

This method isn't called when there is only one LoginForm, this seems to be related to the creation of the tabset?

**Comment by Firesphere** [comment]
I'm working on this by the way. Would it be an asset to have a LoginForm::getTitle method? That would make the process a lot easier!

**Comment by robbieaverill** [comment]
@Firesphere it's worth mentioning that I've just looked at something similar with the active directory module in that `LoginForm` instances now implement a `buildRequestHandler` method which returns an instance of a `FormRequestHandler` class, and that tends to be where processing logic is stored. This isn't very well covered in the changelogs. See the login form and authenticator changes [in this PR](https://github.com/silverstripe/silverstripe-activedirectory/pull/100).

The framework still doesn't allow you to have multiple login forms/authenticators at once though:

```
Fatal error: Class name must be a valid object or a string in .../framework/src/Security/LoginForm.php on line 55
```

**Comment by Firesphere** [comment]
This has been fixed in #6800
